client/landing/gutenboarding/components/domain-picker-button/style.scss

```diff
@@ -7,3 +7,17 @@
 		transform: rotate( 180deg );
 	}
 }
+
+.domain-picker-button__popover {
+	.components-popover__content {
+		border: 1px solid var( --studio-gray-5 );
+		box-shadow: 0 4px 10px rgba( 0, 0, 0, 0.12 );
+		border-radius: 4px;
+	}
+}
+
+.domain-picker-button__popover-container {
+	position: fixed;
+	left: 72px;
+	top: 54px;
```

Comment:
Doesn't the popover component itself take care of this? I sense trouble on mobile. :-)

Reply:
> Doesn't the popover component itself take care of this? I sense trouble on mobile. :-)

Andrew talks about this change [here](https://github.com/Automattic/wp-calypso/pull/40742#issuecomment-609659426):

> * [x] Add a popover container so that the popover can be fixed in position matching the positioning in the Figma — this ensures that the popover doesn't move around as the user selects their domain and the text is updated in the header

I agree with the mobile trouble tho :confused:

Reply:
Aight, sounds like worth figuring out once we get to mobile designs later on. :-)